doc/general/operation-modes/operation-modes.md

```diff
@@ -77,7 +80,7 @@ For the operation of the installed components of the fiskaltrust.Middleware foll
 
 | Hardware                          | Minimum requirements                                         |
 | --------------------------------- | ------------------------------------------------------------ |
-| **General hardware requirements** | The fiskaltrust.Middleware can be generally operated on a [Raspberry PI 2](https://www.raspberrypi.org/products/raspberry-pi-2-model-b/). |
+| **General hardware requirements** | The fiskaltrust.Middleware can be generally operated on a [Raspberry PI 2](https://www.raspberrypi.com/products/raspberry-pi-2-model-b/). |
```

Comment:
```suggestion
| **General hardware requirements** | The fiskaltrust.Middleware can be generally operated on a [Raspberry PI 2](https://www.raspberrypi.com/products/raspberry-pi-2-model-b/). | <!-- markdown-link-check-disable-line -->
```

Maybe the website is blocking the azure ip range or sth like that 🤔
With this comment the check will be disabled for this line.